### DISCUSSION TITLE
No .config

### DISCUSSION THREAD

**Discussion by nithilgadde** [FelixKratz/SketchyBar#319] [Q&A] [answered]
So, I installed sketchybar and I tried accessing my .config directory and sketchybar isn't there. I already had .config for my neovim setup and my sketchybar isn't showing up. Can anyone tell me how to fix this.

**Comment by FelixKratz** [comment, marked as answer]
Right after you install sketchybar you will see a caveats section. There will be some commands listed needed to copy the default config.

**Reply by nithilgadde** [reply]
Thanks! But, I have another question. When I copied your setup, the volume icon isn't loading. How do I fix?

<img width="163" alt="Screenshot 2023-02-03 at 5 46 38 PM" src="https://user-images.githubusercontent.com/110353659/216725376-3cebb8f9-acc6-4574-84a2-054f40101f97.png">

**Reply by FelixKratz** [reply]
Try the newest version of my setup, then the volume should be there: https://github.com/FelixKratz/dotfiles

**Comment by barrettluke** [comment]
I have a mac m1 and tried to follow the install guide and didn't make the sketchybar folder. I tried to build from source and same thing. My solution was to make the folder in my .config with the folder sketchybar and in the folder I copy the folder "plugins" with all of its files . Then I made the file sketchybarrc in the sketchybar folder with all the starter config info. I was going to use what you have in the dotfiles that you mention above but it has a bunch of extra folders that have not been explaind like the items folder, the help folder etc...
